WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -170,7 +170,8 @@ public async Task<CoinJoinResult> StartRoundAsync(IEnumerable<SmartCoin> smartCo
 			var unsignedCoinJoin = await ProceedWithSigningStateAsync(roundId, registeredAliceClients, outputTxOuts, cancellationToken).ConfigureAwait(false);
 
 			var finalRoundState = await RoundStatusUpdater.CreateRoundAwaiter(s => s.Id == roundId && s.Phase == Phase.Ended, cancellationToken).ConfigureAwait(false);
-			Logger.LogDebug($"Round ({roundId}): Round Ended - WasTransactionBroadcast: '{finalRoundState.WasTransactionBroadcast}'.");
+
+			Logger.LogDebug($"Round ({roundId}): Round ended - WasTransactionBroadcast: '{finalRoundState.WasTransactionBroadcast}'.");
```

Comment:
maybe here add txid.

Reply:
+1, but we have a LogCoinJoinSummary method, maybe we could log the TxID there.

Reply:
I opened #7628